tests/base_test.py

```diff
@@ -1,7 +1,30 @@
 import pytest
 import torch
 
-from sbi.inference import SNPE
+from sbi import utils
+from sbi.inference import SNPE, infer
+
+
+def test_infer():
+    # Example is taken from 00_getting_started.ipynb
+    num_dim = 3
+    prior = utils.BoxUniform(low=-2 * torch.ones(num_dim), high=2 * torch.ones(num_dim))
+
+    def simulator(parameter_set):
+        return 1.0 + parameter_set + torch.randn(parameter_set.shape) * 0.1
+
+    posterior = infer(simulator, prior, method="SNPE_A", num_simulations=10)
+    assert posterior is not None, "Most basic use of 'infer' failed"
+    posterior = infer(
+        simulator,
+        prior,
+        method="SNPE_A",
+        num_simulations=10,
+        init_kwargs={'num_components': 5},
```

Comment:
please also test `call_kwargs` (and `train_kwargs` if you add them)

Reply:
can be in the same test

Reply:
and ideally you can set `train_kwargs={"max_num_epochs": 2}`, such that the test is faster